agac_torch/configs/point_maze.yaml

```diff
@@ -0,0 +1,38 @@
+"algorithm":
+    "max_steps": 2000000
+    "eval_freq": 10000
+    "num_epochs": 30
+    "num_episodes_eval": 1
+    "train_freq": 1000
+    "discrete": False
+    "gpu": 0
+"logging":
+    "save_models": False
+    "save_models_freq": 100000
+    "use_neptune": True
+    "experiment_name": "AGAC-PointMaze"
+    "neptune_user_name": "user_name"
+    "neptune_project_name": "project_name"
+    "log_grid": False
+"reinforcement_learning":
+    "actor_learning_rate": 0.0003
+    "critic_learning_rate": 0.0003
+    "adversary_learning_rate": 0.00009
+    "layers_dim": [64, 64]
+    "adv_layers_dim": [64, 64]
+    "discount": 0.99
+    "batch_size": 512
+    "lambda_gae": 0.95
+    "intrinsic_reward_coefficient": 0.0004
+    "entropy_coefficient": 0.0
+    "clipping_epsilon": 0.2
+    "value_loss_clip": 1000
```

Comment:
`value_loss_clip=1000` means no clip, right?

Reply:
Yes you're right ! No the cleanest way to implement it